.github/scripts/run-e2e-tests.sh

```diff
@@ -51,6 +51,6 @@ EOF
 popd
 
 echo "Running tests for environment '$FLC_ENVIRONMENT'..."
-make BRANCH="$TARGET_BRANCH" test/e2e
+make BRANCH="$TARGET_BRANCH" test/e2e-publish
```

Comment:
What happens when a test fails? Does it explicitly say which test failed (printing its name in the summary, etc.), or does it just show the ❌ number increased?

I'm asking because I think this is one of the main values of 'publishing' the test results: having a quick overview of what failed.

Reply:
I think it's okey if we fail a build and publish test results. Not everyone has access to see logs.

Reply:
I noticed that in case of failing installation (operator pods - ErrPullImage) there is no result file.

Reply:
Interesting... IMO let's merge it and get experience on how the published result behaves. There will be room for improvement for sure.